.github/workflows/test.yml

```diff
@@ -7,6 +7,7 @@ env:
   runtime-image-name: ubuntu/dotnet-runtime:test
   aspnet-image-name: ubuntu/dotnet-aspnet:test
   dotnet-version: "6.0"
+  skopeo-image: 'quay.io/skopeo/stable:v1.15.1'
```

Comment:
skopeo-image-name

Reply:
This is not an image name, but an image URI. It should not be named after `skopeo-image-name`.

Reply:
My opinion about this naming is the same as https://github.com/ubuntu-rocks/dotnet/pull/585#discussion_r1665572055. I don't think naming the image URI as `*-name` is a good idea.

Reply:
I see, for this i would wait for the advice of @cjdcordeiro and @rebornplusplus

Reply:
Honestly, I am cool with either -- don't have a strong preference for one or the other.

Reply:
y it's a uri. it's ok like this